src/apps/ontology/get_address.py

```diff
@@ -0,0 +1,25 @@
+from trezor.crypto.curve import nist256p1
+from trezor.messages.OntologyAddress import OntologyAddress
+
+from .helpers import get_address_from_public_key
+
+from apps.common import seed
+from apps.common.layout import show_address, show_qr
+
+
+async def get_address(ctx, msg):
+    address_n = msg.address_n or ()
```

Comment:
ah of course because when this PR was first submitted, it was like this everywhere.

well, please see:
https://github.com/trezor/trezor-core/blob/master/src/apps/ethereum/get_address.py#L12
https://github.com/trezor/trezor-core/blob/master/src/apps/ethereum/address.py#L38